eng/packaging.targets

```diff
@@ -24,7 +24,10 @@
                                        '$(BuildAllConfigurations)' == 'true' and
                                        '$(DotNetBuildFromSource)' == 'true'">true</GeneratePackageOnBuild>
     <!-- Search for the documentation file in the intellisense package and otherwise pick up the generated one. -->
-    <LibIntellisenseDocumentationFilePath>$(XmlDocFileRoot)1033\$(AssemblyName).xml</LibIntellisenseDocumentationFilePath>
```

Comment:
`LibIntellisenseDocumentationFilePath` now gets conditionally defined in two places below.

`IntellisenseNetFile` is always processed, but only once (and used in two places).

`IntellisenseDotNetPlatExtFile` is conditionally processed once, only if we couldn't find the file under `IntellisenseNetFile` (and also used in two places).